pkg/apis/kops/validation/instancegroup.go

```diff
@@ -140,6 +140,8 @@ func ValidateInstanceGroup(g *kops.InstanceGroup, cloud fi.Cloud) field.ErrorLis
 		allErrs = append(allErrs, validateExternalLoadBalancer(&lb, path)...)
 	}
 
+	allErrs = append(allErrs, IsValidValue(field.NewPath("spec", "updatePolicy"), g.Spec.UpdatePolicy, []string{kops.UpdatePolicyAutomatic, kops.UpdatePolicyExternal})...)
```

Comment:
Why not check if `g.Spec.UpdatePolicy != nil`?
Same in other places?

Reply:
That was me using the same `IsValidValue` function that we use [when validating the similar _Cluster_ field](https://github.com/kubernetes/kops/blob/4e8c6558993124af16c6a14efc88548c88daa805/pkg/apis/kops/validation/validation.go#L108), which I used as my example. [`IsValidValue` checks if the pointer argument is nil.](https://github.com/kubernetes/kops/blob/4e8c6558993124af16c6a14efc88548c88daa805/pkg/apis/kops/validation/helpers.go#L38)

Does that address your concern, or have I misunderstood you?

Reply:
Yes. All good. Thanks.